packages/dds/sequence/src/sequence.ts

```diff
@@ -441,17 +441,28 @@ export abstract class SharedSegmentSequence<T extends ISegment>
         return this.intervalMapKernel.keys();
     }
 
-    protected summarizeCore(serializer: IFluidSerializer): ISummaryTreeWithStats {
+    protected summarizeCore(
+        serializer: IFluidSerializer,
+        telemetryContext?: ITelemetryContext,
+    ): ISummaryTreeWithStats {
         const builder = new SummaryTreeBuilder();
 
+        this.incrementSummarizeInstanceCount(telemetryContext);
+
+        let totalBlobBytes = 0;
+
         // conditionally write the interval collection blob
         // only if it has entries
         if (this.intervalMapKernel.size > 0) {
-            builder.addBlob(snapshotFileName, this.intervalMapKernel.serialize(serializer));
+            const content = this.intervalMapKernel.serialize(serializer);
+            totalBlobBytes += content.length;
+            builder.addBlob(snapshotFileName, content);
```

Comment:
This logic duplicates what builder already does. Plus, it does it slightly differently - builder will measure actual size in bytes, assuming utf-8 encoding, whereas this new code measure characters. So, I think at a minimum, we should use data provided by builder.

I'd also look for ways not to need to add this code. Most of our DDSs are based on SharedObject base class, and it can easily capture this kind of high-level data for all of our DDSs (at DDS scope).
DDS may want to collect extra data (which I think you are doing here for interval collection), but I'd rather see this delegated to interval collection itself (i.e. pass telemetryContext to its serialize() method), instead of doing it here.

Reply:
And I think the intention (based on increaseSummarizeTotalBlobBytes call and its implementation below) is to measure whole Sequence DDS, but we capture only part of it (interval collection), skipping main content (summarizeMergeTree call below)